### DISCUSSION TITLE
What happens when an item is not returned in time?

### DISCUSSION THREAD

**Discussion by mimosel** [leihs/leihs#1521] [Q&A / Community Support] [answered]
Since the documentation is unfortunately still offline and I couldn't figure it out with the demo version either, I'm afraid I have to ask again here:

What exactly happens if a borrowed item is not returned in time?

Are there any reminder emails? Is the item automatically blocked for further borrowing by other people? Can overdue fines be managed in leihs? Is there an interface to LBS or FOLIO?

Thanks in advance for your help!
Michael

**Comment by neoheat** [comment, marked as answer]
dear @mimosel, please find our answers here

What exactly happens if a borrowed item is not returned in time?
> - The user will be blocked to borrow any other items in Leihs
> - The user get's notified by E-Mail
> - see https://github.com/leihs/manual/wiki/Abholen%20und%20Zurückbringen#verspätete-rückgabe

Are there any reminder emails?
> - Yes, the user get notified every day about the item which is overdue

Is the item automatically blocked for further borrowing by other people? 
> - Yes, the item is blocked until the user brings back the item. Of course, if there are more then one item of a modell available, there might be another one available. I mention this, because a reservation of an item in Leihs is not made on a specific item (inventory number), it is always an instance of a model.

Can overdue fines be managed in leihs? 
> - It is not managed in Leihs. However, it can be claimed organizationally upon return.

Is there an interface to LBS or FOLIO?
> - we don't have interfaces to LBS or FOLIO
